content/en/security/vulnerabilities/_index.md

```diff
@@ -1,10 +1,8 @@
 ---
 title: Cloud Security Management Vulnerabilities
 kind: documentation
-aliases:
```

Comment:
Looks like this alias was accidentally removed. We'll want to make sure it's re-added to avoid any broken links.

Reply:
That's super strange.. Noted , will add those back (there should be 2)